localization/pt/common.json

```
@@ -1,65 +1,65 @@
-	"overview-button": "See How The Magic Works",
+	"overview-title": "Bem-vindo ao\nVaulst de ~~Yearn~~",
+	"overview-description": "Aqui você pode encontrar informações detalhadas sobre como os yVaults de Yearn operam e as ações (estratégias) que esses vaults executam.\n\nEste site manterá um local único para todas as descrições dos yVaults. Todos os vaults em produção agora são da versão 2 ou superior. Todos os yVaults da versão 1 foram descontinuados.",
+	"overview-button": "Veja Como a Mágica Acontece",
 
-	"yearn-and-curve-eth-vault-title": "Ethereum Vaults",
-	"yearn-and-curve-eth-vault-description": "Yearn Finance started and has the majority of its yVaults running on Ethereum Mainnet. We have deployed over 250 strategies and 100 yVaults on Ethereum. The ones that are currently running in production you can find in the sections below.",
-	"yearn-and-curve-synergy-title": "Yearn and Curve Synergy",
-	"yearn-and-curve-synergy-description": "One of the critical components of Yearn’s infrastructure includes a collaborative relationship with [Curve.fi](https://curve.fi/). Several Yearn vaults provide liquidity into Curve pools and stake the liquidity provider (LP) tokens into the respective gauges, earning CRV rewards.\n\nYearn locks **10%** of all CRV rewards earned into the yveCRV-DAO (“Backscratcher”) to obtain an additional amount of CRV.\n\nIn the strategy descriptions, vaults that are boosted are indicated with a",
-	"yearn-and-curve-synergy-description-next": "\n\nFor a deeper understanding, refer to the [Understanding Curve Boost Multipliers](/curve-boost-multipliers) section.\n\nFurthermore, the remaining 90% of the CRV earned are swapped into the respective LP tokens, and re-deposited into the vault. The only exception is the yvUSDN3Crv vault that locks 50% of the CRV earned into the Backscratcher vault and swaps the remaining 50%.",
+	"yearn-and-curve-eth-vault-title": "Vaults Ethereum",
+	"yearn-and-curve-eth-vault-description": "Yearn Finance começou em, e tem a maioria de seus yVaults rodando na Ethereum Mainnet. Implementamos mais de 250 estratégias e 100 yVaults em Ethereum. Os que estão atualmente em produção você pode encontrar nas seções abaixo.",
+	"yearn-and-curve-synergy-title": "Sinergia Yearn e Curve",
+	"yearn-and-curve-synergy-description": "Um dos componentes críticos da infraestrutura da Yearn inclui um relacionamento colaborativo com [Curve.fi](https://curve.fi/). Vários vaults de Yearn fornecem liquidez aos pools do Curve e fazem stake dos tokens do provedor de liquidez (LP) nos respectivos "gauges", ganhando recompensas em CRV.\n\nYearn bloqueia **10%** de todas as recompensas CRV recebidas no yveCRV-DAO ("Backscratcher") para obter uma quantidade adicional de CRV.\n\nNas descrições da estratégia, os vaults que são impulsionados são indicados com um",
```

Missing some `\` for json parsing

```suggestion
	"yearn-and-curve-synergy-description": "Um dos componentes críticos da infraestrutura da Yearn inclui um relacionamento colaborativo com [Curve.fi](https://curve.fi/). Vários vaults de Yearn fornecem liquidez aos pools do Curve e fazem stake dos tokens do provedor de liquidez (LP) nos respectivos \"gauges\", ganhando recompensas em CRV.\n\nYearn bloqueia **10%** de todas as recompensas CRV recebidas no yveCRV-DAO (\"Backscratcher\") para obter uma quantidade adicional de CRV.\n\nNas descrições da estratégia, os vaults que são impulsionados são indicados com um",
```